packages/flutter/lib/src/widgets/text.dart

```diff
@@ -1228,6 +1226,70 @@ class _SelectableTextContainerDelegate extends MultiSelectableSelectionContainer
     return a.right > b.right ? 1 : -1;
   }
 
+  SelectedContentRange _calculateLocalRange(List<SelectedContentRange> ranges) {
```

Comment:
Nit: Maybe add a comment here. (I also left a comment saying the same thing on the SelectableRegion version of this.)